Sources/App/Features/Events/Models/EventResponse.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  EventResponse.swift
+//  
+//
+//  Created by Alex Logan on 02/08/2022.
+//
+
+import Foundation
+import Vapor
+
+struct EventResponse: Content {
+    var id: UUID?
```

Comment:
I assume we'd always have an id here for the responses (as the data comes form the database), is it possible to make this and other id's mandatory?

Reply:
Whilst this is technically correct to assume that the UUID will always be present, the entities actually have the ID's set as optional ( I believe this is to handle when you've created an entity yet not saved it on the database ).

I've altered the transformers to filter out where ID is nil, so this means we lose the ability to return non-saved entities but that sounds like a problem for another day 😅